install_config/aggregate_logging.adoc

```diff
@@ -1011,6 +1012,121 @@ $ for dc in $(oc get deploymentconfig --selector logging-infra=elasticsearch -o
+===== Configuring hostPath storage for Elasticsearch =====
+
+You can provision {product-title} clusters using hostPath storage for Elasticsearch.
+
+To use a local disk volume on each node host as storage for an Elasticsearch replica:
+
+. Create a local mount point on each infrastructure node for the local Elasticsearch storage:
++
+----
+mkdir /usr/local/es-storage
+----
+
+. Create a filesystem on the Elasticsearch volume:
++
+----
+mkfs.ext4 /dev/xxx
+----
+
+. Mount the elasticsearch volume:
++
+----
+mount /dev/xxx /usr/local/es-storage
+----
+
+. Add the following line to `/etc/fstab`:
++
+----
+/dev/xxx /usr/local/es-storage ext4
+----
+
+. Change ownership for the mount point:
++
+----
+chown 1000:1000 /usr/local/es-storage
+----
+
+. Give the privilege to mount and edit a local volume to the relevant service account:
++
+----
+  $ oc adm policy add-scc-to-user privileged  \
+         system:serviceaccount:logging:aggregated-logging-elasticsearch
+----
+  Use the project you created earlier (for example, logging) when running the logging playbook.
+
+. To claim that privilege, patch each Elasticsearch replica definition, as shown in the example,
+which specifies `--selector component=es-ops` for an Ops cluster:
++
+----
+  $ for dc in $(oc get deploymentconfig --selector component=es -o name);
+do
+    oc scale $dc --replicas=0
+    oc patch $dc \
+       -p '{"spec":{"template":{"spec":{"containers":[{"name":"elasticsearch","securityContext":{"privileged":
+true}}]}}}}'
+done
+----
++
+. Locate the Elasticsearch replicas on the correct nodes to use the local storage,
+and do not move them around, even if those nodes are taken down for a period of time.
+To specify the node location, give each Elasticsearch replica a node selector that is unique
+to a node where an administrator has allocated storage for it.
++
+To configure a node selector, edit each Elasticsearch deployment configuration, adding
+or editing the `nodeSelector` section to specify a unique label that you have applied
+for each node you desire:
++
+----
+apiVersion: v1
+kind: DeploymentConfig
+spec:
+  template:
+    spec:
+      nodeSelector:
+        logging-es-node: "1"
+----
++
+The label must uniquely identify a replica with a single node that bears that label,
+in this case `logging-es-node=1`.
++
+. Create a node selector for each required node. Use the `oc label` command to apply
+labels to as many nodes as needed.
++
+For example, if your deployment has three infrastructure nodes, you could add
+labels for those nodes as follows:
++
+----
+  $ oc label node <nodename1> logging-es-node=1
+  $ oc label node <nodename2> logging-es-node=2
+  $ oc label node <nodename3> logging-es-node=3
+----
++
+To automate application of the node selector, use the `oc patch` command
+instead of the `oc label` command, as follows:
++
+----
+  $ oc patch dc/logging-es-<suffix> \
+     -p '{"spec":{"template":{"spec":{"nodeSelector":{"logging-es-node":"1"}}}}}'
+----
++
```

Comment:
Same note above regarding the extra `+` here.

Reply:
STET, same note as I entered above, too. There is only one "+" in the file, and it didn't seem to break the order of things. So I'm leaving one "+" in the Asciidoc file.